mkosi/installer/apt.py

```diff
@@ -81,6 +81,13 @@ def apt_cmd(context: Context, command: str) -> list[PathString]:
         "-o", "pkgCacheGen::ForceEssential=,",
     ]
 
+    if not context.config.repository_key_check:
+        cmdline += [
+            "-o", "Acquire::AllowInsecureRepositories=true",
+            "-o", "Acquire::AllowDowngradeToInsecureRepositories",
```

Comment:
Receiving the following error with this:
```console
E: Option Acquire::AllowDowngradeToInsecureRepositories: Configuration item specification must have an =<val>.
```

```suggestion
            "-o", "Acquire::AllowDowngradeToInsecureRepositories=true",
```

Reply:
Shame on me for not testing the quick change I made